spec/fixtures/device.conf

```diff
@@ -1,3 +1,9 @@
-[homehub]
+[officehub]
 type philips_hue
-url  file:///Users/david.armstrong/projects/hue_rsapi/spec/fixtures/homehub.conf
\ No newline at end of file
+url  file:///Users/david.armstrong/projects/hue_rsapi/spec/fixtures/officehub.conf
+# Update url so that the path matches your own.
+#  i.e. url  file:///Users/david.swan/GitHub/hue_rsapi/spec/fixtures/officehub.conf
```

Comment:
Is this needed? I get what you're saying but it's just an example with a different hardcoded path

Reply:
Removed